cli_reference/openshift_developer_cli/sample-applications.adoc

```diff
@@ -0,0 +1,125 @@
+[id='odo-sample-applications']
+= Sample applications
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: odo-sample-applications
+toc::[]
+
+
+{odo-title} offers partial compatibility with any language or runtime listed within the OpenShift catalog of component types. For example:
```

Comment:
Instead of copying and pasting this content here, you should just include the `openshift-developer-cli-sample-applications.adoc` module here in the assembly.

You could always title the assembly something like "Using sample applications" to be more consistent with the other assembly titles, and to differentiate it from the module title.

Reply:
Makes sense! I've created an assembly "Using sample applications", moved introduction there. And the original file back to modules.